sympy/solvers/tests/test_solveset.py

```diff
@@ -588,7 +588,7 @@ def test_solveset_complex_polynomial():
 
 def test_sol_zero_complex():
     # This should return the complex set after it is implemented
-    raises(NotImplementedError, lambda: solveset_complex(0, x))
+    assert solveset_complex(0, x) == S.Complex
```

Comment:
I don't really know. It's just a suggestion. I am fine with ``==`` as well.
On 1 Jul 2015 12:36, "AMiT Kumar" <notifications@github.com> wrote:

> In sympy/solvers/tests/test_solveset.py
> <https://github.com/sympy/sympy/pull/9589#discussion_r33653276>:
>
> > @@ -588,7 +588,7 @@ def test_solveset_complex_polynomial():
> >
> >  def test_sol_zero_complex():
> >      # This should return the complex set after it is implemented
> > -    raises(NotImplementedError, lambda: solveset_complex(0, x))
> > +    assert solveset_complex(0, x) == S.Complex
>
> @leosartaj <https://github.com/leosartaj>
> Is there a PEP8 standard for this?
> As far as I know, we normally use is for values like True or False.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/sympy/pull/9589/files#r33653276>.
>